load_balancer.go

```diff
@@ -109,13 +109,42 @@ func (odbi *ovnDBImp) lbAddImp(name string, vipPort string, protocol string, add
 }
 
 func (odbi *ovnDBImp) lbDelImp(name string) (*OvnCommand, error) {
+	var operations []libovsdb.Operation
+
 	condition := libovsdb.NewCondition("name", "==", name)
 	deleteOp := libovsdb.Operation{
 		Op:    opDelete,
 		Table: tableLoadBalancer,
 		Where: []interface{}{condition},
 	}
-	operations := []libovsdb.Operation{deleteOp}
+	operations = append(operations, deleteOp)
+
+	// Also delete references from Logical switches
+	row := make(OVNRow)
+	row["name"] = name
+	lbuuid := odbi.getRowUUID(tableLoadBalancer, row)
+	if len(lbuuid) == 0 {
+		return nil, ErrorNotFound
+	}
+	mutateUUID := []libovsdb.UUID{{lbuuid}}
+	mutateSet, err := libovsdb.NewOvsSet(mutateUUID)
+	if err != nil {
+		return nil, err
+	}
+	mutation := libovsdb.NewMutation("load_balancer", opDelete, mutateSet)
+	ucondition, err := odbi.getRowUUIDContainsUUID(tableLogicalSwitch, "load_balancer", lbuuid)
+	if err != nil && err != ErrorNotFound {
+		return nil, err
+	} else if err == nil {
+		mucondition := libovsdb.NewCondition("load_balancer", "==", libovsdb.UUID{ucondition})
```

Comment:
ucondition is a row uuid in LogicalSwitch table. How come it is compared with "load_balancer" field? Please add test cases to make sure the code actually works as expected.

Reply:
i think that this is error, because getRowUUIDContainsUUID returns uuid of row in logical switch table, so we need to use lbuuid in mutation and delete it from load_balancer row in logical switch table.

Reply:
@hzhou8 corrected and verified. Sorry it was _uuid. PTAL at revised PR.